### PR TITLE
Fix #5998: AppVeyor package building is currently failing

### LINKED ISSUE #5998: AppVeyor package building is currently failing
4 days ago everything was fine, but then the 64 bits package jobs starting failing, now both are failing.
Here are two builds of the same commit showing the difference:
- https://ci.appveyor.com/project/Zimmi48/coq/build/v8.7~119 (4 days ago, fine)
- https://ci.appveyor.com/project/Zimmi48/coq/build/v8.7~129 (today, failing)

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

**Comment by Zimmi48** [comment]
@maximedenes AppVeyor is still failing :'(

File: dev/build/windows/MakeCoq_MinGW.bat [hunk 1]
```diff
@@ -345,7 +345,7 @@ IF "%COQREGTESTING%" == "Y" (
 SET "EXTRAPACKAGES= "
```
Comment (SkySkimmer) [inline dev/build/windows/MakeCoq_MinGW.bat]: This line is very suspicious.

Comment (maximedenes) [reply dev/build/windows/MakeCoq_MinGW.bat]: Really? Why?

**Comment by maximedenes** [comment]
> AppVeyor is still failing :'(

OMG this is probably another Batch substitution I got wrong. Investigating.

File: dev/build/windows/MakeCoq_MinGW.bat (hunk 1 above)
Comment (SkySkimmer) [reply dev/build/windows/MakeCoq_MinGW.bat]: bash instincts I guess

Comment (maximedenes) [reply dev/build/windows/MakeCoq_MinGW.bat]: > bash instincts

would be a nice movie title :)

**Comment by maximedenes** [comment]
OK, there seems to be another, independent problem.

File: dev/ci/appveyor.bat [hunk 2]
```diff
@@ -37,5 +38,5 @@ if %USEOPAM% == true (
 GOTO :EOF
 
 :ErrorExit
-  ECHO ERROR MakeCoq_MinGW.bat failed
+  ECHO ERROR appveyor.bat failed
```
Comment (psteckler) [inline dev/ci/appveyor.bat]: I believe you can use `%0` to get the name of the batch file, and will still work if the file name changes.

**Pull request merged**